### PR TITLE
Code homepage, setup login and register flow, add fonts and style structure.

### PR DESCRIPTION
**Pull request by ceemion**
@SundayAdefila :)

### CODE REVIEW

**Comment by SundayAdefila** [comment]
I'm wise now, I think I know the things to look for. I'll surprise you with this one, I promise. :)

**Comment by SundayAdefila** [comment]
@ceemion  What's the difference between image2 and image22

**Comment by ceemion** [comment]
@SundayAdefila Looool.... img22 is of a higher resolution than img2. I will eventually change the images when I perfect the slider thingy.

File: spec/javascripts/controllers/login_controller_spec.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      $scope = $rootScope.$new();
+      return ctrl = $controller('LoginController', { $scope: $scope });
+    })
+  };
+
+  beforeEach(homeCtrl());
+
+  describe('Initialize login controller', function () {
+    it('sets a test scope', function () {
+      expect($scope.name).toEqual("We're in login");
```
Comment (SundayAdefila) [inline spec/javascripts/controllers/login_controller_spec.js]: When do we start injecting the real code to use and totally do away with the place holders like "We're in login" ?

Comment (ceemion) [inline spec/javascripts/controllers/login_controller_spec.js]: When we start implementing user registration and authentication with devise. Anything else I'll write now will be HTML and CSS with occasional AngularJS.

**Pull request merged**